src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_load_store_emitters.cpp

```diff
@@ -65,18 +62,126 @@ void jit_load_emitter::emit_isa(const std::vector<size_t> &in_idxs, const std::v
     }
 }
 
+template <cpu_isa_t isa>
+void jit_load_emitter::load_dbyte(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const {
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    XReg src = XReg(in_idxs[0]);
+    XReg prc = XReg(aux_gpr_idxs[0]);
+    TReg dst = TReg(out_idxs[0]);
+    DReg dst_d = DReg(out_idxs[0]);
+    HReg dst_h = HReg(out_idxs[0]);
+    SReg dst_s = SReg(out_idxs[0]);
+
+    switch (load_num_) {
+        case 0:
+            break;
+        case 1:
+            h->ldr(dst_h, ptr(src, byte_offset_));
+            break;
+        case 2:
+            h->ldr(dst_s, ptr(src, byte_offset_));
+            break;
+        case 3:
+            h->ldr(dst_s, ptr(src, byte_offset_));
+            h->add_imm(prc, src, byte_offset_ + 2 * sizeof(uint16_t), h->X_DEFAULT_ADDR);
+            h->ld1(dst.h[2], ptr(prc));
+            break;
+        case 4:
+            h->ldr(dst_d, ptr(src, byte_offset_));
+            break;
+        default:
+            OV_CPU_JIT_EMITTER_THROW("Unexpected number of elements to load.");
+    }
+}
+
+template <cpu_isa_t isa>
+void jit_load_emitter::load_byte(const std::vector<size_t> &in_idxs, const std::vector<size_t> &out_idxs) const {
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+    XReg src = XReg(in_idxs[0]);
+    XReg prc = XReg(aux_gpr_idxs[0]);
```

Comment:
If we need to load 4 f32 elements, we don't need to have additional gprs. Moreover, in this case `get_aux_gprs_count()` return 0. But here we use `aux_gpr_idxs[0]`. There will be segfault if `aux_gpr_idxs` is empty.
Could you please define such variables only in used code blocks when we're sure that we need `aux_gpr` and `!aux_gpr_idxs.empty()`?

Reply:
Good point! Applied. Thanks Alexandra!